kasa/tapo/child_device.py

```diff
@@ -0,0 +1,75 @@
+"""Child device implementation."""
+from typing import Dict, Optional, Union
+
+from ..deviceconfig import DeviceConfig
+from ..exceptions import SmartDeviceException
+from ..smartprotocol import SmartProtocol
+from .tapodevice import TapoDevice
+
+
+class ChildProtocolWrapper(SmartProtocol):
+    """Protocol wrapper for controlling child devices.
```

Comment:
I think maybe this should go in `smartprotocol` module.

Reply:
Makes sense, I'll move it :+1: